git-cliff-release/action.yaml

```diff
@@ -33,6 +33,11 @@ inputs:
     required: false
     type: string
     default: ${{ github.token }}
+  no-github:
```

Comment:
I'd try to be more descriptive here - how about `add-github-links` (with `default: true`) for example?